### DISCUSSION TITLE
REST API

### DISCUSSION THREAD

**Discussion by luik8004** [dec0dOS/zero-ui#213] [Q&A]
For example, in my personal account my.zerotier.com to work with the API I use the link https://api.zerotier.com/api/network/181c24c55a85ae4e/member . Let's assume that for zero-ui I use the domain https://zeroui.mydomen.com . Do I understand correctly that when deploying a container I need to specify the variable - ZU_CONTROLLER_ENDPOINT=https://zeroui.mydomen.com and the REST API will be available to me, for example, for the 181c24c55a85ae4e network at https://zeroui.mydomen.com/app/network/181c24c55a85ae4e ?

**Comment by dec0dOS** [comment]
Actually, ZU_CONTROLLER_ENDPOINT is for communicating with the ZeroTier controller (the ZeroTier instance). If you want to bind a specific domain name for access, you should use LISTEN_ADDRESS. Alternatively, you can keep the default value of 0.0.0.0, which allows access from any domain that the reverse proxy permits.
